src/app/layout.tsx

```diff
@@ -27,6 +27,10 @@ export default function RootLayout({
       <head>
         <title>GDI - User Portal</title>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta
```

Comment:
important: it looks like there are more things to configure according to Next.js docs. Can you please check and follow the same solution?

https://nextjs.org/docs/app/building-your-application/configuring/content-security-policy